File: 0782-jewels-and-stones/0782-jewels-and-stones.py
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        count=0
        for cur in stones:
            if cur in jewels:
                count+=1
        return count

File: 0782-jewels-and-stones/0782-jewels-and-stones.c
#include<stdbool.h>
int numJewelsInStones(char* jewels, char* stones) {
   bool hash[256]={false};
   int i,count=0;

   for(i=0;jewels[i]!='\0';i++)
   {
        hash[(unsigned char)jewels[i]]=true;
   }

   for(i=0;stones[i]!='\0';i++)
   {
        if(hash[(unsigned char) stones[i]]==true)
            count++;
   }

   return count;
}
